plugins/edge-config/index.ts

```diff
@@ -10,38 +10,82 @@
   governing permissions and limitations under the License.
 */
 
-import { Configuration } from '@adobe/griffon-toolkit-aep-mobile';
+import { SharedStateConfig } from '@adobe/griffon-toolkit-aep-mobile';
 import { Event } from '@adobe/griffon-toolkit-common';
 import { ValidationPluginResult } from '../../types/validationPlugin';
 
 (function (events: Event[]): ValidationPluginResult {
   const { toolkit: kit } = window.griffon;
-  const { configuration } = kit['aep-mobile'];
+  const { sharedStateConfig } = kit['aep-mobile'];
   const configEvents = kit.match(
```

Comment:
nit: can we rename this configSharedStateEvents not to be confused with other config events